apiserver/facades/client/modelmanager/modelmanager.go

```diff
@@ -872,49 +884,38 @@ func (m *ModelManagerAPI) makeModelSummary(mi state.ModelSummary) *params.ModelS
 		summary.Counts = append(summary.Counts, params.ModelEntityCount{Entity: params.Units, Count: mi.UnitCount})
 	}
 
-	access, err := common.StateToParamsUserAccessPermission(mi.Access)
+	access, err := common.StateToParamsUserAccessPermission(mi.UserAccess)
```

Comment:
We seem to be throwing the error away as well. Is this what we want?

Reply:
Good point, it works as intended with the current implementation of the method, but its pretty terrible practice. I've changed it to be more resilient.